app/controllers/container_dashboard_controller.rb

```diff
@@ -25,10 +27,18 @@ def data_live
     render :json => collect_live_data(params[:id], params[:query])
   end
 
+  def title
+    "container_dashboard"
```

Comment:
`title` should return a human-readable string, wrappd in gettext .. so this should probably be `_("Container Dashboard")` .. thanks :)

(Also, you'll get rid of that code climate warning by moving `private` under `session_key_prefix` ;).)